appinfo/routes.php

```diff
@@ -32,6 +32,14 @@
 				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleStar',
 				'verb' => 'POST'],
 			[
+				'name' => 'messages#toggleUnseen',
+				'url' => '/accounts/{accountId}/folders/{folderId}/messages/{messageId}/toggleUnseen',
+				'verb' => 'POST'],
+			[
+				'name' => 'messages#getUnseenCount',
+				'url' => '/accounts/{accountId}/folders/{folderId}/getUnseenCount',
+				'verb' => 'POST'],
```

Comment:
The very high level design principle is:
- use GET if you want to get anything from the server and the state on the server is not changed. GET is idempotent
- use POST if you want to change a state. POST is not idempotent
- use PUT if you want to set a state on the server. PUT is idempotent